scripts/azure/env.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# get scripts location
+export KUBECONFIG="$( cd "$(dirname "$0")" ; pwd -P )/.kube/config"
```

Comment:
You will also probably need to do something like [this](https://github.com/networkservicemesh/networkservicemesh/blob/master/.env/gke.env) in terms of setting various env variables that will cause the make machinery to use the azure env.

Reply:
This is already [done](https://github.com/networkservicemesh/networkservicemesh/blob/e12916a3be66a9614b34aedf12150859056ce1d2/.env/azure.env)